tf2_ros/CMakeLists.txt

```diff
@@ -182,9 +182,20 @@ add_rostest(test/transform_listener_time_reset_test.launch)
 
 endif()
 
-# Python test
+  # Python test
+  # Provides PYTHON_EXECUTABLE_DEBUG
+  find_package(python_cmake_module REQUIRED)
+  find_package(PythonExtra REQUIRED)
+
+  set(_PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE}")
```

Comment:
We may want to capture/document this bit of logic somewhere, as it will likely come up anytime `pytest` is used on Windows Debug.

Reply:
Yeah, that's a good idea.  Are you thinking of a cmake function/macro, or documentation?  In either case, any thoughts on where we would put it?

Reply:
Not sure. We have a small section on Windows debug in the installation guide, but I don't think that it fits here: https://github.com/ros2/ros2_documentation/blob/master/source/Installation/Eloquent/Windows-Development-Setup.rst#extra-stuff-for-debug-mode

Can it be fixed in ament_cmake_pytest? https://github.com/ament/ament_cmake/blob/master/ament_cmake_pytest/cmake/ament_add_pytest_test.cmake#L61